images/kubekins-e2e/variants.yaml

```diff
@@ -7,26 +7,26 @@ variants:
     UPGRADE_DOCKER: 'true'
   master:
     CONFIG: master
-    GO_VERSION: 1.12.1
+    GO_VERSION: 1.12.7
+    K8S_RELEASE: stable
+    BAZEL_VERSION: 0.23.2
+  '1.16':
+    CONFIG: '1.16'
+    GO_VERSION: 1.12.7
     K8S_RELEASE: stable
     BAZEL_VERSION: 0.23.2
   '1.15':
     CONFIG: '1.15'
-    GO_VERSION: 1.12.1
+    GO_VERSION: 1.12.7
     K8S_RELEASE: stable
     BAZEL_VERSION: 0.23.2
   '1.14':
     CONFIG: '1.14'
-    GO_VERSION: 1.12.1
+    GO_VERSION: 1.12.5
     K8S_RELEASE: latest
```

Comment:
What is the notation used for `K8S_RELEASE`?

Reply:
@Katharine